include/interfaces/non_conservative/navier_stokes.h

```diff
@@ -547,6 +610,31 @@ NavierStokes<dim>::compute_system_operators(const DoFHandler<dim> &dh,
         }
       });
     }
+  else if (prec_name=="elman-2")
+    {
+      auto Ap= linear_operator<TrilinosWrappers::MPI::Vector>(aux_matrices[0]->block(1,1));
+      auto Mp=  linear_operator<TrilinosWrappers::MPI::Vector>(aux_matrices[2]->block(1,1));
+
+      auto Fp =  linear_operator<TrilinosWrappers::MPI::Vector>(aux_matrices[3]->block(1,1));
+
+      Ap_preconditioner->initialize (aux_matrices[3]->block(1,1));
```

Comment:
@asartori86 ho un segfault in questo punto. Non riesco a reperire ulteriori dati...
Ho provato a mettere anche jacobi anziché AMG ma il segfault persiste.... sto sbagliando qualche cosa?